bootstrap/cmd/bootstrap/app/server.go

```diff
@@ -538,20 +554,19 @@ func Run(opt *options.ServerOption) error {
 			rawCmd := fmt.Sprintf(
 				"ks apply default -c %v --token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)",
 				component.Name)
-			applyCmd := exec.Command("bash", "-c", rawCmd)
-
-			var out bytes.Buffer
-			var stderr bytes.Buffer
-			applyCmd.Stdout = &out
-			applyCmd.Stderr = &stderr
-			if err := applyCmd.Run(); err != nil {
-				log.Infof("Component apply failed: " + fmt.Sprint(err) + "\n" + out.String() + "\n" + stderr.String())
-				return err
-			} else {
-				// ks apply output to stderr on success case as well
-				log.Infof("Component apply successfully.\n" + out.String() + "\n" + stderr.String())
-			}
+
+      if err := runBashCommand(rawCmd, "Component apply"); err != nil {
+        return err
+      }
 		}
+
+    // Deploy GKE driver daemonset.
+    err := runBashCommand(
+      "kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/stable/nvidia-driver-installer/cos/daemonset-preloaded.yaml",
```

Comment:
Can we make this a K8s package/component that the bootstrapper installs?

Reply:
Do you mean make it into a ksonnet component as part of kubeflow package?

Reply:
Yes. This way the daemonset becomes part of the ks app so you aren't dependent on bootstrapper in order to have all the configs needed for the cluster.